src/components/CreateBook.js

```diff
@@ -28,28 +28,38 @@ const CreateBook = () => {
 
   return (
     <div>
-      <h2>Add New Book</h2>
-      <div className="form">
-        <input
-          className="input"
-          type="text"
-          placeholder="Book title"
-          value={book.title}
-          name="title"
-          onChange={handleChange}
-        />
-        <input
-          type="text"
-          placeholder="Author name"
-          value={book.author}
-          name="author"
-          onChange={handleChange}
-        />
-
-        <button className="button" onClick={onSaveBookClicked} type="button">
-          Submit
-        </button>
+      <div className="bookform">
+        <h1 className="heading">Create Book</h1>
+        <div className="input">
+          <input
+            className="titleinput"
+            type="text"
+            placeholder="Enter book name"
+            name="title"
+            value={book.title}
+            onChange={handleChange}
+          />
+          <select
+            className="cat"
+            name="author"
+            onChange={handleChange}
+          >
+            {categories.map((option) => (
+              <option key={option} value={option}>
+                {option}
+              </option>
+            ))}
+          </select>
+          <button
+            type="submit"
+            className="btnsubmit"
+            onClick={onSaveBookClicked}
+          >
+            Add Book
+          </button>
+        </div>
       </div>
+
     </div>
```

Comment:
- [ ] You're doing well with your project, but ensuring that your form component's layout aligns with the design is essential. To achieve this, please update your code to match the design's specifications, including proper padding, margin, and spacing.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/BilalLiaquat7/bookstore/assets/55840999/99362060-03c1-453a-a51d-738e87628c1e)| ![image](https://github.com/BilalLiaquat7/bookstore/assets/55840999/32854bd4-717a-47a7-93a8-61655c2ac44d)|


By making these adjustments, you'll ensure that your form component follows the design closely, resulting in a more visually consistent and polished user interface. Keep up the great work! 🌟🎨